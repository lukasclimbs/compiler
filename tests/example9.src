if true then true else  3
