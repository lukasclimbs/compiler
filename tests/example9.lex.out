[(,if,true,true,3,),]
